include/armadillo_bits/def_lapack.hpp

```diff
@@ -23,7 +23,11 @@
 #endif
 
 #if !defined(ARMA_BLAS_CAPITALS)
-  
+  #define arma_sgehrd sgehrd
```

Comment:
the uppercase versions of the lapack names are also needed; see the defines after the #else statement